pkg/services/provisioning/plugins/plugin_provisioner.go

```diff
@@ -52,7 +52,7 @@ func (ap *PluginProvisioner) apply(ctx context.Context, cfg *pluginsAsConfig) er
 			return errors.New("plugin not found")
 		}
 		if p.AutoEnabled && !app.Enabled {
-			return errors.New("plugin is auto enabled and cannot be disabled")
```

Comment:
Maybe we can improve the error message here to point to `[plugins.disable_plugins]`. Something like:

```
			return errors.New("app is auto enabled and cannot be disabled through provisioning: use [plugins.disabled_plugins] configuration instead")
```